packages/plugin-analytic-google/src/GoogleAnalytics4.ts

```diff
@@ -47,6 +48,8 @@ export class GoogleAnalytics4 extends AbstractAnalytic {
 
   /**
    * Initializes the Google Analytics 4 plugin.
+   * @param config
```

Comment:
we can remove these params from jsdoc, there is no added value..

Reply:
(After discussion I disabled the rules and removed the defs.)